### DISCUSSION TITLE
Using PEM certificates

### DISCUSSION THREAD

**Discussion by exussum12** [nginx-proxy/nginx-proxy#2442] [Ideas]
Currently the template is looking for a .crt and a .key file, would there be any interest in PEM support?

**Comment by buchdag** [comment]
Hi.

While that might sound like a nice small QoL improvement at first glance, I'm actually not sure it would really be worth the additional template code complexity. If you're able to put a `.pem` file inside the container's `/etc/nginx/certs` folder, you're inevitably also able to put a `.crt` file in the same place with the same content instead.

**Reply by exussum12** [reply]
Yeah the template was the worry and hence raising this rather than sending a PR.

The use case was for tools which output pems (eg mkcert), as you say, they can be translated to a .crt and a .key instead though
